Extensions/PrimitiveDrawing/Extension.cpp

```diff
@@ -392,6 +423,36 @@ void DeclarePrimitiveDrawingExtension(gd::PlatformExtension& extension) {
       .SetFunctionName("SetOutlineColor")
       .SetIncludeFile("PrimitiveDrawing/ShapePainterObject.h");
 
+  obj.AddExpression(
+        "GetOutlineColorR",
+        _("Outline color red component"),
+        _("Current value for red outline colors."),
```

Comment:
Same thing for all of these, let's put the same name and description. 
It's too much work to translate different descriptions for something that is almost similar to the name.